member-service/src/main/resources/application.yml

```
@@ -18,6 +32,13 @@ eureka:
-  expiration-time: 86400000 # 1days
\ No newline at end of file
+management:
+  endpoints:
+    web:
+      exposure:
+        include:
+          - refresh
+          - health
+          - beans
+          - info
+          - busrefresh
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)